src/watcher.js

```diff
@@ -143,7 +143,7 @@ function Watcher(bot, league) {
             self.bot.say({
                 text: "<@" + white + ">,  <@" + black + ">:"
                     + " Your game is *not valid* because "
-                    + "*" + result.reason + "*",
+                    + "*" + result.reason + "*" + "@seb32",
```

Comment:
Also, this won't work unless you have notifications on for strings matching your name.

You need to use the user ID and Mark up to identify it as a special token.